integration_tests/test_grpc.py

```diff
@@ -1,11 +1,15 @@
 import json
 
-from pystarport.proto_python.grpc_util import GrpcUtil, GrpcUtilTxBroadcast
+import pytest
 
 from .utils import wait_for_new_blocks
 
+pytestmark = pytest.mark.grpc
+
 
 def test_query_validators(cluster):
+    from pystarport.proto_python.grpc_util import GrpcUtil
```

Comment:
move it here so when proto files not prepared locally, it don't prevent other tests from running.